DEVELOPMENT.md

```diff
@@ -102,7 +102,7 @@ argocd login argocd-server-argocd.apps.127.0.0.1.nip.io:8443
 
 Argo CD web UI is accessible at <https://argocd-server-argocd.apps.127.0.0.1.nip.io:8443/applications>.
 
-GitOps is the preferred approach for deploying the Pipeline Service. The installed instances of ArgoCD can be leveraged for creating organisations in kcp, universal workspaces used by the infrastructure, installing the Tekton controllers and registering the workload clusters.
+GitOps is the preferred approach for deploying the Pipeline Service. The installed instances of Argo CD can be leveraged for creating organisations in kcp, universal workspaces used by the infrastructure, installing the Tekton controllers and registering the workload clusters.
```

Comment:
`organisations` --> kcp uses the term `organization`.

Reply:
what if you want to create more than one organization? :P

Reply:
if you're talking about the 's' and the 'z' - its not a big deal. Just a different form of the same word (US vs UK based English)

Reply:
> if you're talking about the plural form of the word - what if you want to create more than one organization? :P

ah, I meant `organizatoins` ;), missed `s`